caom2utils/setup.cfg

```diff
@@ -33,7 +33,7 @@ url = https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/caom2
 edit_on_github = False
 github_project = opencadc/caom2tools
 # version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
-version = 1.7.3
+version = 1.7.4
```

Comment:
Probably need to update the `cadcutils` dependency to make sure it picks up the one with replace disabled.

Reply:
Good catch.